python/lsst/summit/utils/nightReport.py

```diff
@@ -536,7 +537,8 @@ def getExposureMidpoint(self, seqNum):
             The midpoint, as an mjd float.
         """
         timespan = self.data[seqNum]['timespan']
-        return (timespan.begin.mjd + timespan.end.mjd) / 2
+        expTime = self.data[seqNum]['exposure_time']
+        return ((timespan.begin) + expTime / 2).to_datetime()
```

Comment:
If these are coming from the butler, I would expect the only difference between new and old to be for readouts with significant dark time (which IIRC is included in `timespan` but not `exposure_time`).  Is that actually why you changed this?

Reply:
As discussed OOB, this was designed to have thee same value, but be a python datetime rather than a float.